multichain-testing/Makefile

```
@@ -61,7 +61,7 @@ setup-kind:
 
 .PHONY: clean-kind
 clean-kind:
-	kind delete cluster --name $(KIND_CLUSTER)
+	-kind delete cluster --name $(KIND_CLUSTER)
```

Is this intentional? What does it do?

---

Triggers a silent failure in case the target fails avoiding disruptively halting the Makefile.

---

TIL!

Is this PR still draft?

---

Oops. Let me make it R4R.